### PR TITLE
[PFC WD]: Initial support

### PR DESCRIPTION
**Pull request by marian-pritsak**

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -40,10 +41,16 @@ namespace swss {
...
 
 #define COUNTERS_PORT_NAME_MAP          "COUNTERS_PORT_NAME_MAP"
+#define COUNTERS_TABLE                  "COUNTERS"
 
 #define DAEMON_TABLE_NAME "DAEMON_TABLE"
 #define DAEMON_LOGLEVEL "LOGLEVEL"
 
+#define APP_PFC_WD_TABLE_NAME "PFC_WD_TABLE"
+#define PFC_WD_STATE_TABLE "PFC_WD_STATE_TABLE"
+#define PFC_WD_PORT_COUNTER_ID_LIST "port_counter_id_list"
```
Comment (stcheng) [inline common/schema.h]: capitalize them?

Comment (marian-pritsak) [reply common/schema.h]: Done

**Comment by stcheng** [comment]
@sihuihan88 please check.

File: common/redisapi.h [hunk 2]
```diff
@@ -61,4 +63,69 @@ static inline std::string loadLuaScript(const std::string& path)
...
+        auto ctx = r.getContext();
+        SWSS_LOG_DEBUG("Running lua script %s", sha.c_str());
+
+        if (ctx->type == REDIS_REPLY_NIL)
+        {
+            SWSS_LOG_ERROR("Got EMPTY response type from redis %d", ctx->type);
+            return std::move(ret);
+        }
+
+        else if (ctx->type != REDIS_REPLY_ARRAY)
```
Comment (lguohan) [inline common/redisapi.h]: no space between the bracket and else if.

**Comment by lguohan** [comment]
can you regenerate the pr against master?

**Pull request closed without merging**